src/graph/backend/dnnl/kernels/sdp_decomp.hpp

```diff
@@ -163,6 +163,7 @@ struct sdp_decomp_kernel_t : public kernel_base_t {
         UNUSED(outputs);
         UNUSED(sycl_deps);
         UNUSED(sycl_event);
+        VINFO(graph, exec, check, sdp, "no sycl execution for decomp sdp ");
```

Comment:
I feel we don't need these VINFO check in this file.

Reply:
removed. thanks for your advice